common/templates/includes/footer-merchant-details.njk

```diff
@@ -0,0 +1,8 @@
+ {% if service and service.merchantDetails %}
+   <ul class="merchant-details">
+     <li class="merchant-details-line-1">Service provided by {{ service.merchantDetails.name }}</li><br/>
+     <li class="merchant-details-line-2">{{ service.merchantDetails.addressLine1 }},{% if service.merchantDetails.addressLine2 %} {{ service.merchantDetails.addressLine2 }},{% endif %} {{ service.merchantDetails.city }} {{ service.merchantDetails.postcode }} {{ service.merchantDetails.countryName }}</li><br/>
+   <li class="merchant-details-phone-number">Phone: {{ service.merchantDetails.phoneNumber }}</li>
+   <li class="merchant-details-email">Email: {{ service.merchantDetails.email }}</li>
+   </ul>
```

Comment:
I am not sure using a `<ul>` for an address is symantically correct, could just use a `<p>` or you could _even_ use `<address>` — https://iandevlin.com/blog/2012/01/html/marking-up-a-postal-address-with-html/ — TLDR, scroll to the bottom

Reply:
Ah, cool! I have just done it the way it is done in the prototype / frontend. Let's take a look.

Reply:
Apparently the gov.uk template does not allow anything outside of `<ul>` `/confused`. Will merge and then fix later, in case